config/service_providers.yml

```diff
@@ -21,6 +21,11 @@ development:
       sp_initiated_login_url: 'http://localhost:3000/test/saml'
       cert: 'sample_cert'
       fingerprint: 'F9:A3:9B:2F:8F:1C:E2:79:27:69:EB:32:ED:2A:D5:A2:A7:58:5F:C0:74:8A:4A:03:D9:0F:77:A5:89:7F:F9:68'
+    'urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost':
+      metadata_url: 'http://localhost:4567/saml/metadata'
+      acs_url: 'http://localhost:4567/consume'
+      sp_initiated_login_url: 'http://localhost:3000/test/saml'
```

Comment:
This will be on the same host as the other URLs. the default with omniauth will be `/users/auth/saml`